dotcom-rendering/src/web/layouts/LiveLayout.tsx

```diff
@@ -748,6 +749,20 @@ export const LiveLayout = ({ CAPI, NAV, format, palette }: Props) => {
 										/>
 									</div>
 								</GridItem>
+								<GridItem area="matchstats">
+									{CAPI.matchUrl && (
```

Comment:
I really like how we have variants of the grid type, it helped me understand how the match stats are separated from the standard grid.

Could matchUrl ever be an empty string? In StandardLayout I think we account for this by `!!CAPI.matchUrl` - it might be something we consider here